openfl/experimental/workflow/interface/cli/workspace.py

```diff
@@ -54,6 +54,7 @@ def create_dirs(prefix):
     (prefix / "data").mkdir(parents=True, exist_ok=True)  # training data
     (prefix / "logs").mkdir(parents=True, exist_ok=True)  # training logs
     (prefix / "save").mkdir(parents=True, exist_ok=True)  # model weight saves / initialization
+    (prefix / "local_state").mkdir(parents=True, exist_ok=True)  # persistent state
```

Comment:
The Workflow API's `FederatedRuntime` uses a different [Aggregator class](https://github.com/securefederatedai/openfl/blob/develop/openfl/experimental/workflow/component/aggregator/aggregator.py#L25), which needs to be additionally updated to support restarts. 

For now, I suggest not to change anything in this area, and focus on TaskRunner API.

Reply:
Agree - this change should not be made for the workflow API for now